upb/msg.c

```diff
@@ -285,6 +285,7 @@ static int _upb_mapsorter_cmpstr(const void *_a, const void *_b) {
   return a.size - b.size;
 }
 
+#include <stdio.h>
```

Comment:
this feels weird

Reply:
Residual printf debugging artifact, removed.